base/initdefs.jl

```diff
@@ -278,6 +278,11 @@ function load_path_expand(env::AbstractString)::Union{String, Nothing}
 end
 load_path_expand(::Nothing) = nothing
 
+"""
+    active_project()
+
+Return the path of the active `Project.toml` file.
```

Comment:
```suggestion
The path of the active `Project.toml` file.
```
See style guide https://docs.julialang.org/en/v1/manual/documentation/#Documentation

Reply:
What part of the style guide suggests this?

Reply:
> Include a single one-line sentence describing what the function does or what the object represents after the simplified signature block. If needed, provide more details in a second paragraph, after a blank line.
> 
> The one-line sentence should use the imperative form ("Do this", "Return that") instead of the third person (do not write "Returns the length...") when documenting functions. It should end with a period. If the meaning of a function cannot be summarized easily, splitting it into separate composable parts could be beneficial (this should not be taken as an absolute requirement for every single case though).

Reply:
But that block says that `Return ...` is correct?

> The one-line sentence should use the imperative form ("Do this", "Return that")

Reply:
No worries! Done.